### DISCUSSION TITLE
Armenian glyphs

### DISCUSSION THREAD

**Discussion by bobbbay** [fabrizioschiavi/pragmatapro#324] [Ideas]
Heyo!

Just bought the first font I've ever paid for, and I really am loving Pragmata Pro - except for the fact that there are no Armenian glyphs. If these were supported, I'd finally be able to set this as my system font :D.

**Comment by fabrizioschiavi** [comment]
Don't tempt me 😉 I might design the Armenian glyphs the next time…
